### PR TITLE
Compile PROPACK with OpenBLAS on Windows platforms

### PR DESCRIPTION
**Pull request by amontoison**
`LBT_jll` is not working on Windows platforms.

### CODE REVIEW

File: P/PROPACK/build_tarballs.jl [hunk 1]
```diff
@@ -43,8 +50,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363")),
```
Comment (giordano) [inline P/PROPACK/build_tarballs.jl:53]: I'd filter the dependencies by platform

File: P/PROPACK/build_tarballs.jl [hunk 2]
```diff
@@ -43,8 +50,9 @@ products = [
...
     Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"))
```
Comment (amontoison) [inline P/PROPACK/build_tarballs.jl:54]:
```suggestion
    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), platforms=!Sys.iswindows(platforms))
```

File: P/PROPACK/build_tarballs.jl (hunk 1 above)
Comment (amontoison) [inline P/PROPACK/build_tarballs.jl:53]:
```suggestion
    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"), platforms=Sys.iswindows(platforms)),
```

File: P/PROPACK/build_tarballs.jl [hunk 3]
```diff
@@ -43,8 +50,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"))
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"), platforms=Sys.iswindows(platforms)),
```
Comment (amontoison) [inline P/PROPACK/build_tarballs.jl:53]:
```suggestion
    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"), platforms=Sys.iswindows.(platforms)),
```

File: P/PROPACK/build_tarballs.jl [hunk 4]
```diff
@@ -43,8 +50,9 @@ products = [
...
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), platforms=!Sys.iswindows(platforms))
```
Comment (amontoison) [inline P/PROPACK/build_tarballs.jl:54]:
```suggestion
    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), platforms=!Sys.iswindows.(platforms))
```

File: P/PROPACK/build_tarballs.jl [hunk 5]
```diff
@@ -43,8 +50,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"))
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"), platforms=Sys.iswindows.(platforms)),
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), platforms=!Sys.iswindows.(platforms))
```
Comment (amontoison) [inline P/PROPACK/build_tarballs.jl:54]:
```suggestion
    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"), platforms=filter(Sys.iswindows, platforms)),
    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"), platforms=filter(!Sys.iswindows, platforms))
```

**Pull request merged**